public/frontend/src/components/rec-resource/section/Contact.tsx

```diff
@@ -39,7 +39,17 @@ const Contact = forwardRef<HTMLElement, SiteOperatorProps>(
                     Time.
                   </p>
                   <p>
-                    <a href={getContactEmailLink(rec_resource)}>Contact Us</a>
+                    <Link
+                      to={{
+                        pathname: ROUTE_PATHS.REC_RESOURCE_CONTACT.replace(
+                          ':id',
+                          rec_resource_id,
+                        ),
+                        hash: '#contact-us',
+                      }}
+                    >
+                      Contact Us
+                    </Link>
```

Comment:
Using Link is just a nit and non blocker, though as I said in the other comment the effect of scrolling up and appearing on another page was unexpected. If Harry asked for this disregard my comments.

```suggestion
<a
    href={ROUTE_PATHS.REC_RESOURCE_CONTACT.replace(
      ':id',
      rec_resource_id,
    )}
  >
    Contact Us
 </a>

```